homeassistant/components/home_connect/coordinator.py

```diff
@@ -592,3 +602,62 @@ async def update_options(
                 [],
             ):
                 listener()
+
+    def refreshed_too_often_recently(self, appliance_ha_id: str) -> bool:
+        """Check if the appliance data hasn't been refreshed too often recently."""
+
+        now = datetime.now()
```

Comment:
Maybe not used datetimes and timedelta for this time keeping? It's enough to have a simple float timestamp I think. That's less expensive to create.